src/main/java/io/appium/java_client/AppiumBy.java

```diff
@@ -257,4 +323,56 @@ protected ByIosNsPredicate(String locatorString) {
             super("-ios predicate string", locatorString, "iOSNsPredicate");
         }
     }
+
+    public static class FlutterBy extends AppiumBy {
```

Comment:
Can it be made abstract since its not going to be instantiated directly?

Reply:
Agree, will make the class abstract!